nomad/structs/structs.go

```diff
@@ -4720,6 +4734,7 @@ type JobListStub struct {
 	ModifyIndex       uint64
 	JobModifyIndex    uint64
 	SubmitTime        int64
+	Meta              map[string]string
```

Comment:
```suggestion
	Meta              map[string]string `json:",omitempty"`
```

Welcome the API side of things, this is where we make the life of UI engineers harder 😅 

I think omitting the `Meta` field if not requested makes sense? Otherwise the job list will always return `null`:

```console
$ nomad operator api '/v1/jobs?pretty'
[
  {
    "CreateIndex": 11,
    ... 
    "Meta": null,
    "ModifyIndex": 22,
    "Multiregion": null,
    ...
  }
]
``` 

But I don't know if this makes things harder for the UI since the field may not be present if `Meta` is empty.

Reply:
Good catch, @lgfa29

Reply:
Ahhhh this is awesome and will definitely simplify the work we have to do in JS, as "property exists but is null" is treated differently than "property doesn't exist" (Ember will overwrite existing meta details in the former case).

TIL omitempty!